RELEASES.md

```diff
@@ -57,6 +58,109 @@ Get acquainted with the latest EDP releases.
 * [Version 2.6.0](#2.6.0)
 </details>
 
+## Version 3.10.0 <a name="3.10.0"></a> (October 8, 2024)
+
+## What's New
+
+These release notes start with introducing Bitbucket Git server support. This feature enriches the users' toolset and makes the platform more attractive for Bitbucket users.
+
+As a mean of adapting to the uproar the revolves around Artificial Intelligence, we equip KubeRocketCI with a [chat assistant](https://docs.kuberocketci.io/docs/next/user-guide/add-ai-assistant) support feature.
+By integrating chat assistants with the platform, users can get an intelligent assistant which can provide them with informational support.
+
+A new look comes with a new release. Meet our new [documentation site](https://docs.kuberocketci.io/). This site was reworked in a way to match the style of KubeRocketCI portal.
+Besides, the new version boasts documentation versioning and improved navigation.
+
+To improve user experience and clarity in your CI/CD flows, we significantly redesigned the Environment creation procedure. To start from, we updated terminology within the platform.
+From now on, Environments become Deployment Flows and Stages are now Environments. This renaming was done for better reflecting the purpose of each entity.
+Additionally, Deployment Flows and Environments are now created separately and have new settings to apply configurable pipelines.
+
+Regarding of pipelines, this release also presents major enhancements to pipelines. To address users' specific needs, we are working on making pipelines more customizable.
+Now users can make up their own build, review, or deploy pipelines and easily apply them to their when both creating and editing Codebases and Environments.
```

Comment:
'Now users can make up their own build, review, or deploy pipelines' - previously we also have this feature, this feature more about flexible and easy approach how to define pipelines and customization